src/lake_modelling/utils/user_inputs.py

```diff
@@ -0,0 +1,69 @@
+import streamlit as st
+
+
+def get_initial_cond():
+    st.markdown("### Initial Conditions")
+    col1, col2 = st.columns(2)
+    C_lake0 = col1.number_input(
+        "Initial lake Ca concentration (mg-Ca/l)", min_value=0, value=1
+    )
+    C_in0 = col1.number_input(
+        "Lake inflow Ca concentration (mg-Ca/l)", min_value=0, value=1
+    )
+    C_bott0 = col2.number_input(
+        "Lake bottom concentration (mg-Ca/l)", min_value=0, value=0
+    )
+    pH_lake0 = col2.number_input("Lake pH", min_value=1.0, max_value=7.0, value=4.5)
+
+    init_cond = C_lake0, C_in0, C_bott0, pH_lake0
+
+    return init_cond
+
+
+def get_lake_params():
+    st.markdown("### Lake characteristics")
+    col1, col2 = st.columns(2)
+
+    area = col1.number_input("Lake area (km2)", min_value=0.0, value=1.14)
+    mean_depth = col1.number_input("Mean lake depth (m)", min_value=0.0, value=5.6)
+    res_time = col2.number_input("Volume / mean_annual_q", min_value=0, value=10)
```

Comment:
Just checking, is there a reason why this field is "forced" to be integer? Setting `min_value` to 0 instead of 0.0 will always round the provided value to integer, so there is no option to set the residence time to be e.g. 0.7. Shouldn't we allow for decimal input here as well?

Reply:
Yep, `res_time` should accept a float.

I think the label should probably be something like "Water residence time (years)" too.

For the moment, I think default values for lake characteristics should be:

- Lake area: 0.2 km2
- Residence time: 0.7 years
- Mean depth: 5 m
- Flow profile: `none`